todoapp/todos/views.py

```
@@ -26,28 +26,22 @@ class TodoAPIViewSet(rest_viewsets.ModelViewSet):
     ]
     """
 
-    permission_classes = [rest_permissions.AllowAny]
+    authentication_classes = [rest_authentication.TokenAuthentication]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)